content/groth16/en/groth16.md

```diff
@@ -299,10 +299,10 @@ The equation above is valid, but the witness does not necessarily satisfy the or
 
 Therefore, we need to prevent the prover from using $\Psi_1$ to $\Psi_{\ell}$ as part of the computation of $[C]_1$.
 
-### Introducing $\gamma$ and $\delta$
-To avoid the problem above, the trusted setup introduces new scalars $\gamma$ and $\delta$ to force $\Psi_{\ell+1}$ to $\Psi_m$ to be separate from $\Psi_1$ to $\Psi_{\ell}$. To do this, the trusted setup divides (multiplies by the modular inverse) the private terms (that constitute $[C]_1$) by $\delta$ and the public terms (that constitute $[X]_1$, the sum the verifier computes) by $\gamma$.
+### Introducing $\gamma$ and/or $\delta$
+To avoid the problem above, the trusted setup introduces a new scalar : $\gamma$ or $\delta$ to force $\Psi_{\ell+1}$ to $\Psi_m$ to be separate from $\Psi_1$ to $\Psi_{\ell}$. To do this, the trusted setup divides (multiplies by the modular inverse) the private terms (that constitute $[C]_1$) by $\delta$ and/or the public terms (that constitute $[X]_1$, the sum the verifier computes) by $\gamma$.
 
-Since the $h(\tau)t(\tau)$ term is embedded in $[C]_1$, those terms also need to be divided by $\delta$.
+Since the $h(\tau)t(\tau)$ term is embedded in $[C]_1$, those terms also need to be divided by $\delta$. If either $\delta$ and $\gamma$ have an unknown discrete logarithm, then the forgery described earlier along possible other methods are avoided. This method is typically used in Zcash’s Sapling’s‑based trusted setups where $\gamma$ is simply $G_2$ and $\delta$ is still randomly generated.
```

Comment:
"typically" isn't the right word because it was either used or it wasn't. I think "This method was used in Zcash's Sapling's-based [trusted setups](https://github.com/ebfull/phase2/blob/master/src/lib.rs#L808)"

we need to include the hyperlink because we can't expect the reader to look it up themselves.

Reply:
@RareSkills : done.